test_scenarios/T3_Create_New_CT.md

```diff
@@ -0,0 +1,40 @@
+# Test Scenario
+
+## T1 Stworzenie nowego CT
+
+### Sprawdzane UC
+
+- [UC6](../scenarios/UC6_Create%20CT)
+
+### Dane początkowe
+
+![model](data/T3_in.svg)
+
+### Warunki początkowe 
+
+- Zalogowany __App User__ to _user01_.
+
+### Przypadki testowe
+
+#### 1. Stworzenie nowego CT
+
+1. __App User__ wybiera przycisk "Add Computation Task"
```

Comment:
Lista aplikacji jest w innym UC.

Reply:
1. App User wybiera aplikację z listy w Computation Cockpit <<*trigger*>>
2. Launcher wyświetla szczegóły aplikacji <<*screen*>>
3. App User wybiera opcję "Create CT" <<*trigger*>>

Reply:
Owszem, w innym UC, ale scenariusz testowy nie musi się zapinać na UC. Przemyślę to jeszcze i pogadam z Krzyśkiem o ew. przeniesieniu tych kroków do warunków początkowych.

Reply:
To jest w Requirements

Reply:
A warunki początkowe to User jest w Computation Cockpit, oraz jakaś zdefiniowana Aplikacja początkowa by się przydała.

Reply:
@krzysztofpcy a ty co myślisz?

Reply:
Zmieniłem.

Reply:
Library może być potem rozszerzone o query do pobierania aplikacji, jakieś stronnicowanie itd. Założyliśmy teraz prostszą wersję, póki aplikacji jest mało (<100)